main.py

```diff
@@ -26,59 +31,79 @@ def validate_number(num):
         return False
 
 
-def validate_operator(operator):
+def validate_operator(operator: any) -> bool:
+    """Validate value for being in a list of operators and print a massage if the validation failed."""
     if operator in valid_operators:
         return True
 
     print(f'Please enter one on these operators: {", ".join(valid_operators)}')
     return False
 
 
-def handle_input(phrase, validator):
+def handle_input(phrase: str, validator: Callable[[any], bool]) -> T:
+    """
+    Wait for user's input, exit app or validate and return input
+
+    :param phrase: guiding message for the user
+    :param validator: function to validate user input
+    :return: valid user input
+    """
     res = input(phrase)
 
     if res == 'exit':
         app_exit()
-        return
 
     if not validator(res):
         res = handle_input(phrase, validator)
 
     return res
 
 
-def inputs():
+def read_user_inputs() -> None:
+    """Ask user to enter 2 numbers and operators, process them and show result, rinse and repeat"""
     a = handle_input("Enter first number:", validate_number)
     operator = handle_input("Enter the operator you want to use:", validate_operator)
     b = handle_input("Enter second number:", validate_number)
     count(a, b, operator)
+    read_user_inputs()
+
 
+def count(a: T, b: T, operator: str) -> None:
```

Comment:
this is a place when we can remove this type T. Note that when you're calling count() method, you're passing 3 arguments: a,b,operator that are all strings. No need to say that 'a' can be int or float. The same for b.
replace 'T' with 'str' here also